ansible/roles/prosody/files/mod_muc_moderators.lua

```diff
@@ -272,6 +276,15 @@ function process_lobby_muc_loaded(lobby_muc, host_module)
         local main_room = lobby_room.main_room;
         local has_persistent_lobby = main_room._data.starts_with_lobby;
         module:log("debug", "Occupant %s joined lobby room", event.occupant.jid);
+        -- moderators from main room will also join the lobby room when created
+        -- this makes sure to not forward the event to webhooks
+        -- updates the moderator_occupants_in_lobby list
+        if main_room:get_affiliation(event.occupant.bare_jid) == 'owner' or event.occupant.role == "moderator" then
+            table.insert(moderator_occupants_in_lobby, event.occupant.bare_jid);
+        else
+            module:log("debug", "Trigger join lobby room webhook for occupant %s", event.occupant.jid);
+            module:fire_event("muc-occupant-joined", event);
```

Comment:
Make sure you don't fire the internal prosody events as unpredicted results may occur.